docs/diagrams/ThemeCommandSequenceDiagram.puml

```diff
@@ -33,10 +33,6 @@ ThemeFactory --> ThemeCommand : theme
 
 ref over ThemeCommand, ThemeFactory : ThemeManager#setTheme(theme, themePath)
 
-ThemeCommand -> ThemeManager : applyThemeToScene()
```

Comment:
Where is the new `ThemeCommandSequenceDiagram.png`?

Reply:
It's updated as a binary file with the same file name.

Reply:
Oops, my bad.